nmigen/hdl/xfrm.py

```diff
@@ -164,6 +176,14 @@ def on_value(self, value):
     def on_Assign(self, stmt):
         return Assign(self.on_value(stmt.lhs), self.on_value(stmt.rhs))
 
+    def on_Assert(self, stmt):
+        # Preserve _en, as the compiler state thinks _en is driven.
```

Comment:
This comment is incorrect; `_en` *is* in fact driven. Besides, preserving parts of statements during transforms doesn't really need additional comments.